package.json

```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "request": "^2.79.0",
-    "stream-json": "1.3.1",
+    "stream-json": "0.6.1",
```

Comment:
Newer stream-json doesn't passes position info., it is essential for LSP error highlighting.